MonoGame.Framework/GameWindow.cs

```diff
@@ -16,12 +16,10 @@ public abstract class GameWindow {
 
 		public abstract Rectangle ClientBounds { get; }
 
-	    internal bool _allowAltF4 = true;
-
         /// <summary>
         /// Gets or sets a bool that enables usage of Alt+F4 for window closing on desktop platforms. Value is true by default.
         /// </summary>
-        public virtual bool AllowAltF4 { get { return _allowAltF4; } set { _allowAltF4 = value; } }
+        public virtual bool AllowAltF4 { get; set; }

```

Comment:
One less variable to think about, `_allowAltF4` is really only used for the property and nothing else.